Cargo.toml

```diff
@@ -581,7 +581,7 @@ egui_tiles = { git = "https://github.com/rerun-io/egui_tiles", rev = "48e0ef5664
 # walkers = { git = "https://github.com/rerun-io/walkers", rev = "8939cceb3fa49ca8648ee16fe1d8432f5ab0bdcc" } # https://github.com/podusowski/walkers/pull/222
 
 # commit on `rerun-io/re_arrow2` `main` branch
-# https://github.com/rerun-io/re_arrow2/commit/e4717d6debc6d4474ec10db8f629f823f57bad07
-# re_arrow2 = { git = "https://github.com/rerun-io/re_arrow2", rev = "e4717d6debc6d4474ec10db8f629f823f57bad07" }
+# https://github.com/rerun-io/re_arrow2/commit/61c97ff8fb117c2aafa72a4989fa262ed7951c3d
+re_arrow2 = { git = "https://github.com/rerun-io/re_arrow2", rev = "61c97ff8fb117c2aafa72a4989fa262ed7951c3d" }
```

Comment:
Waiting for https://github.com/rerun-io/re_arrow2/pull/12 to be merged so we can point to a commit on `main` instead